core/src/main/java/org/arquillian/smart/testing/SystemPropertyResolver.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.smart.testing;
+
+public class SystemPropertyResolver {
```

Comment:
`System.getProperty(GitRunnerProperties.FOO_BAR)` is effectively the same as `SystemPropertyResolver.getGitFooBar`, but latter has disadvantages which I shared above.